scripts/qa_crowbarsetup.sh

```diff
@@ -4116,12 +4116,21 @@ function oncontroller_testsetup
     fi
 
     # this file is created by the designate barclamp
-    designate_pools="/etc/desigate/pools.crowbar.yaml"
-    [[ -e $designate_pools ]] && designate-manage pool update --file $designate_pools
+    designate_pools="/etc/designate/pools.crowbar.yaml"
+    if [[ -e $designate_pools ]]; then
+        designate-manage pool update --file $designate_pools
+        # default values from the node.neutron.dns_domain
+        openstack tld create --name local
+        openstack zone create --email admin@example.org openstack.local.
+        theport=$(openstack port create --network fixed --dns-name iamgroot thanos -f value -c id)
+        openstack floating ip create floating --port $theport
+        wait_for 30 5 "nslookup iamgroot.openstack.local >> /dev/null" "thanos port is nslookup-able"
+    fi
 
     # Run Tempest Smoketests if configured to do so
     tempestret=0
     if [[ $want_tempest = 1 ]]; then
+        openstack tld create --name com # needed by designate-template-plugin smoke test
```

Comment:
What if you don't have designate deployed ? won't this command fail ?

Reply:
I tested this without designate deployed and it worked, but I don't understand which service responds to this CLI request without designate deployed.

Reply:
thats true, let me fix that